Documentation/guides/messages/xa0127.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Xamarin.Android error XA0127
+description: XA0127 error code
+ms.date: 08/07/2020
+---
+# Xamarin.Android error XA0127
+
+## Issue
+
+If you encounter this issue something unexpected happened with the Fast Deployment
+native tooling. This is most likely to be a native crash.
```

Comment:
Maybe an example error message would be good here?

Reply:
I've never managed to get it to crash so I'm not sure lol